modules/cli/src/main/scala/scala/cli/launcher/LauncherCli.scala

```diff
@@ -22,7 +22,7 @@ object LauncherCli {
 
     val logger          = LoggingOptions().logger
     val cache           = CoursierOptions().coursierCache(logger.coursierLogger(""))
-    val scalaVersion    = options.cliScalaVersion.getOrElse(Properties.versionNumberString)
+    val scalaVersion    = scalaCliVersion(version, options.cliScalaVersion)
```

Comment:
imo it's more readable to leave it as: 
```suggestion
    val scalaVersion    = options.cliScalaVersion.getOrElse(scalaCliVersion(version))
```
no point to pass `options.cliScalaVersion` to the method, since if it's defined, we're taking the value as is.